examples/datadog-agent-agent-apm.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: datadoghq.com/v1alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog
+spec:
+  credentials:
+    apiKey: <DATADOG_API_KEY>
+    appKey: <DATADOG_APP_KEY>
+  agent:
+    image:
+      name: "datadog/agent:latest"
+    config:
+      logLevel: "DEBUG"
```

Comment:
do we need this? the default is INFO in the helm chart

Reply:
Probably not, I took the existing example as-is. Good catch.